src/applications/edu-benefits/feedback-tool/config/form.js

```diff
@@ -18,20 +18,21 @@ import SchoolSelectField from '../components/SchoolSelectField.jsx';
 import GetFormHelp from '../../components/GetFormHelp';
 
 import {
-  transform,
-  submit,
-  recordApplicantRelationship,
-  recruitingLabel,
   accreditationLabel,
+  changeInDegreeLabel,
+  creditTransferLabel,
   financialIssuesLabel,
-  studentLoansLabel,
+  gradePolicyLabel,
   jobOpportunitiesLabel,
-  changeInDegreeLabel,
   qualityLabel,
-  gradePolicyLabel,
+  recordApplicantRelationship,
+  recruitingLabel,
+  refundIssuesLabel,
+  studentLoansLabel,
+  submit,
   transcriptReleaseLabel,
-  creditTransferLabel,
-  refundIssuesLabel
+  transform,
+  validateMatch,
```

Comment:
Hmmm... https://eslint.org/docs/rules/sort-imports

Reply:
We don't have that rule enabled, likely because we tend to organize our import according to their path.

Reply:
Yeah, I was just checking it out and didn't like it. I just want to be able to enforce the sort order of the named imports. And the sort order of deconstructed variables, too, which is a different thing.